electrum/bitcoin.py

```diff
@@ -752,6 +753,36 @@ def is_private_key(key: str, *, raise_on_error=False) -> bool:
             raise
         return False
 
+def parse_script(x):
+    script = ''
+    for word in x.split():
+        if word[0:3] == 'OP_':
+            opcode_int = opcodes[word]
+            script += construct_script([opcode_int])
+        else:
+            bfh(word)  # to test it is hex data
+            script += construct_script([word])
+    return script
+
+def parse_address(line):
+    r = line.strip()
+    m = re.match('^'+RE_ALIAS+'$', r)
```

Comment:
`RE_ALIAS` is not defined here.
(but see other comment first: don't move `parse_address` to bitcoin.py at all)